### PR TITLE
[Do Not Merge] Announce NRQL Results Limit Increases

### PR DESCRIPTION
**Pull request by ethanbeaver**
This PR Adds a What's New announcement doc for announcing the increase of NRQL Results limits. This is changing from 2000 to 5000, and this should announce that change to customers.

### CODE REVIEW

File: src/content/whats-new/2023/11/whats-new-11-07-nrql-limit-increases.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: 'NRQL Query Result Limits are Increasing'
+summary: 'The maximum result limit for NRQL queries is increasing from 2000 to 5000'
+releaseDate: '2023-11-07'
+learnMoreLink: 'https://docs.newrelic.com/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions/#sel-limit'
+---
+
+
+NRQL Query Result Limits are increasing! When querying New Relic with NRQL, the `LIMIT` clause is used to specify the maximum number of results to be returned from a query. The maximum allowable value has been 2000, and this is increasing to a value of 5000. This increase will allow you to get more from your NRQL Queries in your dashboards, Query Builder, and more.
```
Comment (m0rticus) [inline src/content/whats-new/2023/11/whats-new-11-07-nrql-limit-increases.md:9]:
```suggestion
The previous maximum of 2000 results is now increasing to a new maximum of 5000. 
```

**Comment by rhetoric101** [comment]
I send a DM to @ethanbeaver to clarify the status of this PR.

**Comment by maddie-boisvert** [comment]
Hi @rhetoric101! I believe Ethan is still out for the holidays, so this PR can be closed in favor of https://github.com/newrelic/docs-website/pull/15730 which supersedes these changes.

**Comment by akristen** [comment]
Closing this PR per comments in the conversation tab + this PR: https://github.com/newrelic/docs-website/pull/15730

**Pull request closed without merging**